sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -454,17 +453,10 @@ def test_transform():
         voting='soft',
         flatten_transform=False).fit(X, y)
 
-    warn_msg = ("'flatten_transform' default value will be "
-                "changed to True in 0.21. "
-                "To silence this warning you may"
-                " explicitly set flatten_transform=False.")
-    res = assert_warns_message(DeprecationWarning, warn_msg,
-                               eclf1.transform, X)
-    assert_array_equal(res.shape, (3, 4, 2))
+    assert_array_equal(eclf1.transform(X).shape, (4, 6))
     assert_array_equal(eclf2.transform(X).shape, (4, 6))
     assert_array_equal(eclf3.transform(X).shape, (3, 4, 2))
-    assert_array_almost_equal(res.swapaxes(0, 1).reshape((4, 6)),
```

Comment:
Should  ideally keep a check that values are identical between eclf1 and 2

Reply:
I thought shapes were enough but sure, can add that back in.

Reply:
Yes, we rarely check that the defaults are what's expected, but it doesn't hurt to do so.